src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,116 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.DataBaseConnector;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        try (Connection connection = DataBaseConnector.getConnection(); PreparedStatement
+                statement = connection.prepareStatement((Query.INSERT.string),
+                        Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, manufacturer.getName());
+            statement.setString(2, manufacturer.getCountry());
+            statement.executeUpdate();
+            ResultSet generatedKeys = statement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1, Long.class);
+                manufacturer.setId(id);
+            } else {
+                throw new DataProcessingException("Insert error!", new Throwable());
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create manufacturer!", e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        try (Connection connection = DataBaseConnector.getConnection(); PreparedStatement statement

```

Comment:
```suggestion
        try (Connection connection = DataBaseConnector.getConnection(); 
```
let place each resource on the new line

Reply:
Maven checkstyle demands extra whitespaces, that don't offered by IDEA, so it looks strange to me.